README.md

```diff
@@ -45,7 +45,7 @@ $( selector ).imageLightbox({
     caption:        false,                   // bool;            show captions
     enableKeyboard: true,                    // bool;            enable keyboard shortcuts (arrows Left/Right and Esc)
     history:        false,                   // bool;            enable image permalinks and history
-    fullscreen:     false                    // bool;            enable fullscreen (enter/return key)
+    fullscreen:     false,                    // bool;            enable fullscreen (enter/return key)
```

Comment:
Hi, thanks for the PR! Could you please fix the alignment of the comments? I'll merge it then.

Reply:
done :)